package.json

```diff
@@ -15,6 +15,7 @@
     "gulp-babel": "^8.0.0",
     "gulp-clean-css": "^3.7.0",
     "gulp-concat": "^2.6.1",
+    "gulp-css-url-custom-hash": "0.0.5",
```

Comment:
```suggestion
    "gulp-css-url-custom-hash": "^0.0.5",
```

Reply:
I just ran `$ npm install gulp-css-url-custom-hash --save` but it didn't add the prefix. Maybe because there's only one version? 🤔 I'm not sure.